setup.cfg

```diff
@@ -9,3 +9,8 @@ directory=build/reports/coverage
 
 [coverage:xml]
 output=build/reports/coverage.xml
+
+[flake8]
```

Comment:
This is already configured in `.prospector.yaml`, and you are setting a different max-line-length.

Reply:
I'll remove this